packages/sdk/package.json

```diff
@@ -2,86 +2,68 @@
   "name": "@hop-protocol/sdk",
   "version": "0.0.1-beta.593",
   "description": "The Hop Protocol JavaScript SDK",
+  "author": "Authereum Labs, Inc.",
+  "license": "MIT",
   "main": "dist/src/index.js",
   "types": "dist/src/index.d.ts",
-  "engines": {
-    "node": ">=14"
-  },
   "scripts": {
     "test": "jest",
-    "dev": "npm run version && npx -p typescript tsc --watch",
+    "dev": "npm run version && tsc --watch",
     "build": "npm run version && npm run build:ts",
-    "build:ts": "ls -l node_modules | grep @hop; ls -l ../../node_modules | grep @hop; npx -p typescript tsc",
+    "build:ts": "ls -l node_modules | grep @hop; ls -l ../../node_modules | grep @hop; tsc",
     "build:browser": "browserify -t [ babelify --presets [ @babel/preset-env ] --plugins [ @babel/plugin-transform-runtime ] ] dist/src/index.js | uglifyjs --compress --mangle > hop.js",
     "clean": "rimraf dist && rimraf hop.js",
     "clean:docs": "rimraf docs",
     "clean:docs-ui": "rimraf docs-ui",
     "version": "echo \"export default '$(cat package.json | jq -r '.version')'\" > src/version.ts",
-    "lint": "npx eslint --fix \"src/**/*\"",
+    "lint": "eslint --fix",
     "version:bump": "npm version prerelease --preid=beta",
     "publish:latest": "npm publish --tag latest --access public",
     "publish:beta": "npm publish --tag beta --access public",
     "publish:next": "npm publish --tag next --access public",
     "dist-tag:latest": "npm dist-tag add \"@hop-protocol/sdk@$(jq -r .version <package.json)\" latest",
     "dist-tag:beta": "npm dist-tag add \"@hop-protocol/sdk@$(jq -r .version <package.json)\" beta",
     "dist-tag:next": "npm dist-tag add \"@hop-protocol/sdk@$(jq -r .version <package.json)\" next",
-    "version:sync": "package-sync-version",
```

Comment:
by removing this, how do the other packages get updated with the latest version of this package?

Reply:
@miguelmota sorry I must have resovled this one without replying.

Updated all local deps to use the pnpm `workspace:*` [syntax](https://pnpm.io/workspaces#workspace-protocol-workspace). This resolves to the real numbers when published.

That means we just need to bump the `package.json` version instead of updating every file. We already do this with the `pnpm bump`, which uses native `pnpm` pogic.